File: Backend/Backend_ProyectoFinal/Backend_ProyectoFinal/Models/Usuario.cs
namespace Backend_ProyectoFinal.Models
{
    public class Usuario
    {
        private int idUsuario;
        private string user;
        private string contrasenia;
        private string nombre;
        private string apellido;
        private string tipoUsuario;

        public Usuario(int id, string user, string contra, string nombre, string apellido, string tipoUsuario)
        {
            this.idUsuario = id;
            this.user = user;
            this.contrasenia = contra;
            this.nombre = nombre;
            this.apellido = apellido;
            this.tipoUsuario = tipoUsuario;
        }

        public Usuario()
        {
        }

        public int IdUsuario { get => idUsuario; set => idUsuario = value; }
        public string User { get => user; set => user = value; }
        public string Contrasenia { get => contrasenia; set => contrasenia = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellido { get => apellido; set => apellido = value; }
        public string TipoUsuario { get => tipoUsuario; set => tipoUsuario = value; }
    }
}

File: ProyectoFinal/src/main/java/modelos/Usuario.java
package modelos;

public class Usuario {

    String user;
    String contrasenia;
    String nombre;
    String apellido;
    String correo;
    String tipoUsuario;

    public Usuario(String user, String contrasenia, String nombre, String apellido, String correo, String tipoUsuario) {
        this.user = user;
        this.contrasenia = contrasenia;
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo = correo;
        this.tipoUsuario = tipoUsuario;
    }

    public Usuario() {
    }

    public Usuario(String user, String contrasenia, String nombre, String apellido, String correo) {
        this.user = user;
        this.contrasenia = contrasenia;
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo = correo;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getContrasenia() {
        return contrasenia;
    }

    public void setContrasenia(String contrasenia) {
        this.contrasenia = contrasenia;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getTipoUsuario() {
        return tipoUsuario;
    }

    public void setTipoUsuario(String tipoUsuario) {
        this.tipoUsuario = tipoUsuario;
    }
}
